ui/lib/core/addon/components/form-field.hbs

```diff
@@ -16,13 +16,13 @@
             <G.Legend data-test-form-field-label>{{this.labelString}}</G.Legend>
           {{/if}}
           {{#if this.helpTextString}}
-            <G.HelperText data-test-help-text>{{this.helpTextString}}</G.HelperText>
+            <G.HelperText data-test-form-field-help-text>{{this.helpTextString}}</G.HelperText>
           {{/if}}
           {{#if @attr.options.subText}}
-            <G.HelperText data-test-label-subtext>
+            <G.HelperText data-test-form-field-subtext>
```

Comment:
This is where things get a little hard, but while this is "subtext" in the component, it's still using the helper text hds component. Thus, sticking with the pattern, we should still use the helpText selector we made above, but modify what we pass in so we can select it in the relevant test.
```suggestion
            <G.HelperText data-test-form-field-subtext>
```
```suggestion
            <G.HelperText data-test-help-text={{@attr.options.subText}}=>
```